wp/wp-content/themes/lahma_maker/js/scripts.js

```diff
@@ -291,16 +291,58 @@ $( document ).ready(function() {
     jQuery(document).on('click', '#sortreverse', function(e) {
```

Comment:
Isn't it misleading naming that we use an ID `sortreverse` here and `alphabetical` for the other button? Couldn't we use a more unified naming (e.g., `sort_on_date` and `sort_on_abc`).

Reply:
ok, it stayed from the older, renaming makes sense